contrib/completions/bash/openshift

```diff
@@ -13883,6 +13883,77 @@ _openshift_ex_sync-groups()
     must_have_one_noun=()
 }
 
+_openshift_ex_prune-groups()
+{
+    last_command="openshift_ex_prune-groups"
+    commands=()
+
+    flags=()
+    two_word_flags=()
+    flags_with_completion=()
+    flags_completion=()
+
+    flags+=("--blacklist=")
+    flags+=("--confirm")
+    flags+=("--sync-config=")
+    flags+=("--whitelist=")
+    flags+=("--alsologtostderr")
+    flags+=("--api-version=")
+    flags+=("--boot-id-file=")
+    flags+=("--certificate-authority=")
+    flags_with_completion+=("--certificate-authority")
+    flags_completion+=("_filedir")
+    flags+=("--client-certificate=")
+    flags_with_completion+=("--client-certificate")
+    flags_completion+=("_filedir")
+    flags+=("--client-key=")
+    flags_with_completion+=("--client-key")
+    flags_completion+=("_filedir")
+    flags+=("--cluster=")
+    flags+=("--config=")
+    flags_with_completion+=("--config")
+    flags_completion+=("_filedir")
+    flags+=("--container-hints=")
```

Comment:
I don't know what this is and there doesn't seem to be anything wrong / different about the `cobra